spring-reactive-modules/spring-5-reactive-client-2/src/main/resources/streamlargefile/run.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+MYSELF="$(readlink -f "$0")"
+MYDIR="${MYSELF%/*}"
+
+client="${1:-Large}"
+url="${2:-http://localhost:8081/large-file}"
+download_destination="${3:-/tmp/download.dat}"
+xmx="${4:-32m}"
+
+module_dir="$(readlink -f "$MYDIR/../../../..")"
+
+echo "module: $module_dir"
+cd $module_dir
```

Comment:
A good practice when you use cd in a script is to validate you are in the correct directory or exit, for instance:
`cd $module_dir || exit`